src/shared/locales/en/hooks.json

```
@@ -52,5 +52,11 @@
   },
   "useGetFullTransactions": {
     "formatExtendedDate": "dd MMM yyyy"
+  },
+  "useAuthentication": {
+    "errors": {
+      "unauthorizedAction": "Unauthorized. Please try again or contact support.",
+      "invalidPassword": "Error verifying password. Please try again."
+    }
```

Use and replicate in the others translations:
```suggestion
    "unauthorizedError": "Unauthorized. Please try again or contact support.",
    "invalidPasswordError": "Error verifying password. Please try again."
```